lib/languages/ex.js

```diff
@@ -2,10 +2,8 @@
  * Elixir
  */
 module.exports = {
-    // Find document blocks in heredocs that are arguments of the @apidoc
-    // module attribute. Elixir heredocs can be enclosed between """ and """ or
-    // between ''' and '''. Heredocs in ~s and ~S sigils are also supported.
-    docBlocksRegExp: /@apidoc\s*(~[sS])?"""\uffff?(.+?)\uffff?(?:\s*)?"""|@apidoc\s*(~[sS])?'''\uffff?(.+?)\uffff?(?:\s*)?'''/g,
-    // Remove not needed tabs at the beginning
-    inlineRegExp: /^(\t*)?/gm
+    // Find document blocks between '#{' and '#}'
+    docBlocksRegExp: /\#*\{\uffff?(.+?)\uffff?(?:\s*)?\#+\}/g,
```

Comment:
Elixir 1.6 formatter will insert a space between # and space (i.e. "#{" => "# {"), can we update the regex to allow a space and merge it?